packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -74,7 +74,7 @@ export function useFrequentlyBoughtTogether<TObject>({
           setStatus('loading');
         },
         onResult(response) {
-          setResult(response as GetRecommendationsResult<TObject>);
+          setResult((response as unknown) as GetRecommendationsResult<TObject>);
```

Comment:
@francoischalifour reset the state here to reflect your feedback, but i'm not super happy with the type. Any ideas on how we can improve this?  cc @marialungu